net/cloudflared/files/cloudflared.config

```diff
@@ -1,9 +1,6 @@
-
```

Comment:
please leave these blank lines as-is. even you removed them here, they will be automatically added by uci